v2_papers/2010/lim_internet.json

```diff
@@ -0,0 +1,287 @@
+{
+  "version": "v2.1",
+  "reference": {
+    "title": "Internet Traffic Classification Demystified: On the Sources of the Discriminative Power",
+    "authors": [
+      "Lim, Yeon-sup",
+      "Kim, Hyun-chul",
+      "Jeong, Jiwoong",
+      "Kim, Chong-kwon",
+      "Kwon, Ted Taekyoung",
+      "Choi, Yanghee"
+    ],
+    "publication_name": "Co-NEXT '10 Proceedings of the 6th International COnference",
+    "publication_type": "peer_reviewed_conference",
+    "year": 2010,
+    "organization_publishers": [
+      "acm"
+    ],
+    "pages_number_of": 12,
+    "bibtex": {
+      "type": "inproceedings",
+      "booktitle": "Proceedings of the 6th International COnference",
+      "organization": "ACM"
+    },
+    "access_open": true,
+    "curated_by": "Meghdouri, f.",
+    "curated_last_revision": "10-04-2018",
+    "curated_revision_number": 3
+  },
+  "data": {
+    "datasets": [
+      {
+        "name": "PAIX",
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured",
+        "generation_year": 2004,
+        "covered_period": "hours",
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "anonymized": true
+      },
+      {
+        "name": "WIDE-I",
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured",
+        "generation_year": 2006,
+        "covered_period": "minutes",
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "anonymized": true
+      },
+      {
+        "name": "WIDE-II",
```

Comment:
Regarding WIDE-II (and I assume WIDE-I is the same), there are two versions:
1. one that is anonymized, without payloads, and public (http://imdc.datcat.org/collection/1-055M-0=WIDE-TRANSIT+100+Megabit+Ethernet+Trace+2007-01-09+%28Anonymized%29)
2. one that is not anonymized, has the first 96 bytes of each packet, and is available on request (http://imdc.datcat.org/collection/1-055N-M=WIDE-TRANSIT-100-Megabit-Ethernet-Trace-2007-01-09)

Since they don't cite the datasets it's hard to find out which version it is exactly, but I'd say it's the first one since they require the payloads. If you agree, change ``availability`` to ``public_on_demand`` and ``anonymized`` to ``false``.
I guess for WIDE-I it's exactly the same, but I'm only able to find the public version (anonymized, without payloads): http://mawi.wide.ad.jp/mawi/samplepoint-B/20060303/

Reply:
I took the informations from the paper and I didn't check from the website, I'll do the changes